src/components/Articles/Articles.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import ArticleCard from "../ArticleCard/ArticleCard"
+import "./Articles.css"
+
+function AllArticles() {
+	const [articles, setArticles] = useState([]);
+  const [articlesLoading, setArticlesLoading] = useState(true)
+
+	useEffect(() => {
+    setArticlesLoading(true)
+		axios
```

Comment:
It would be good to separate out the API calls into functions in a separate file and import them in where necessary throughout your project to keep your components tidy 🧹